Files/Strings/ru-RU/Resources.resw

```diff
@@ -34,7 +34,7 @@
     <value>Очистить все</value>
   </data>
   <data name="RecentItemDescription.Text" xml:space="preserve">
-    <value>Файлы с которыми вы ранее взаимодействовали будут отображаться здесь</value>
+    <value>Файлы, с которыми вы ранее взаимодействовали, будут отображаться здесь</value>
```

Comment:
Please, don't commit any change to the resw file and remove the already made changes, the compilation of this file will be made in a future commit by a member of the Team. You only have to take care of the xlf file.